src/Components/LocationAutocompleteInput.tsx

```diff
@@ -103,6 +103,7 @@ export const LocationAutocompleteInput: FC<LocationAutocompleteInputProps> = ({
 
   const handleSuggestionsFetchRequested = useMemo(
     () => debounce(updateSuggestions, DEBOUNCE_DELAY),
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
It was brought up recently not to disable this error unless absolutely necessary. A solution could be to wrap `updateSuggestions` in a useCallback hook.

Reply:
Updated